jwst/refpix/reference_pixels.py

```diff
@@ -1230,25 +1211,31 @@ def do_fullframe_corrections(self):
                     self.group = thisgroup
                 #
                 #  Now transform back from detector to DMS coordinates.
-                self.detector_to_DMS(integration, group)
+                self.detector_to_dms(integration, group)
         log.setLevel(logging.INFO)
         return
 
     def do_subarray_corrections(self):
-        """Do corrections for subarray.  Reference pixel value calculated
-        separately for odd and even columns if odd_even_columns is True,
-        otherwise a single number calculated from all reference pixels"""
+        """
+        Do corrections for subarrays.
+
+        Reference pixel values are calculated separately for odd and even columns
+        if odd_even_columns is True, otherwise a single number calculated from
+        all reference pixels
+        """
         #
         #  First transform to detector coordinates
         #
```

Comment:
The style checker doesn't flag these but I admit it's a pet peeve of mine to have commented empty lines like this. 
 Ultimately it's up to you if you want to remove them, but I would be happy if you did

Reply:
Fair enough, I don't think the whitespace adds anything to comprehension